test/integration/test_win_group3.yml

```diff
@@ -9,3 +9,4 @@
     - { role: win_command, tags: test_win_command }
     - { role: win_reg_stat, tags: test_win_reg_stat }
     - { role: win_region, tags: test_win-region }
+    - { role: win_firewall_rule, tags: test_win_firewall_rule }
```

Comment:
I don't think you have to make this change yourself. Just adding it to the right group as done above is sufficient IIRC.